src/services/ledger/paginated-ledger.ts

```diff
@@ -0,0 +1,73 @@
+// The paginatedLedger function is a copy of medici's MainBook.ledger function
+// but without the page/perPage logic and with new args first/last
+// that paginate the database response based on the transaction _id field
+
+// This should be used for any list of transactions that's exposed in the API
+
+import { ClientSession, Types } from "mongoose"
+import { IAnyObject } from "medici"
+
+import { transactionModel } from "medici/build/models/transaction"
+import { parseFilterQuery } from "medici/build/helper/parse/parseFilterQuery"
+
+import { MainBook } from "./books"
+
+export const DEFAULT_MAX_CONNECTION_LIMIT = 100
+
+interface ITransaction {
```

Comment:
please remove this and use ILedgerTransaction https://github.com/GaloyMoney/galoy/blob/main/src/services/ledger/schema.types.d.ts#L1